lib/osf-components/addon/components/file-renderer/component.ts

```diff
@@ -42,7 +44,6 @@ export default class FileRenderer extends Component {
     download?: string;
     lastDownload?: string;
     width = '100%';
-    height = '100%';
```

Comment:
I don't think removing the default height definition here and then specifying the height in css as `90vh` is a good way to go. Since the default height is `100%`, that means it should always take the full height of it's parent element if the parent element has `height` definied. Otherwise, it would trace all the way up until there is an ancestor element with defined `height`.

That means, instead of specifying the height of the iframe directly, you could apply a `height` definition to its parent div to control the height of the iframe. I believe this is the better way since we know that the iframe will fill the parent element unless specified otherwise.

Reply:
Ok, would you like to keep the default? It had been causing issues with the footer when we had discussed it. When locally testing, only by removing the default and applying height directly to the iframe in the SCSS file seemed to resolve it. If you like, we can pair and go over other options.